src/Microsoft.VisualStudio.Threading.Tests/TplExtensionsTests.cs

```diff
@@ -19,7 +19,9 @@ public TplExtensionsTests(ITestOutputHelper logger)
         [Fact]
         public void CompletedTask()
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
💡 Just mark the test obsolete

Reply:
I know I could do that, but the test isn't obsolete. I feel weird about doing that (or seeing it later).